docs/dev/debug_delve.md

```diff
@@ -0,0 +1,91 @@
+
+# assisted-service runs on minikube
+
+## Debug Setup
+
+To debug assisted-service directly from minikube container
+Install minikube setup with assisted-service with one replica
+
+```shell
+export KUBECONFIG=source /root/.kube/config
+kubectl patch deployment assisted-service -n assisted-installer --type json -p="[{\"op\": \"remove\", \"path\": \"/spec/template/spec/containers/0/livenessProbe\"}]"
+kubectl patch deployment assisted-service -n assisted-installer --type json -p="[{\"op\": \"remove\", \"path\": \"/spec/template/spec/containers/0/readinessProbe\"}]"
+oc rollout restart deployment -n assisted-installer assisted-service
+```
+
+Login to minikube node and attach to assisted-service container
+
+```shell
+oc rsh -n assisted-installer [pod/assisted-service-name] 
+# install packages git goland and telnet
+dnf install procps psmisc golang delve iproute iputils telnet net-tools git -y
```

Comment:
Is there any scenario where this is desirable over using the debug [image](https://github.com/openshift/assisted-service/blob/master/Dockerfile.assisted-service-debug)?

Reply:
The only case i can think about is existnig minikue setup and we want to jump in and attach to golang binary to debug ... it might be service but also from agent too on hosts .
Just have a setup and want to get hands dirty quickly .... of course when we reboot the pod all installed stuff deleted....

anyway its up to you , i used to work with python pdb and asked to do the same with golang :) remote ide is also great but you will have to do all preperation before ....

Reply:
@paul-maidment  Adding Paul about image apprach . also we may have a seperate flow for image